client/my-sites/checkout/composite-checkout/hooks/use-create-payment-complete-callback.tsx

```diff
@@ -332,7 +332,7 @@ function displayRenewalSuccessNotice(
 				{
 					args: {
 						productName: renewalItem.product_name,
-						duration: moment.duration( { days: renewalItem.bill_period } ).humanize(),
+						duration: moment.duration( { days: parseInt( renewalItem.bill_period ) } ).humanize(),
```

Comment:
missing radix here as well. this is a good opportunity to clean up a likely bug while we're inspecting it